examples/high_zoom_map_style.js

```diff
@@ -33,6 +33,9 @@ var mapStyle = {
       "service",
       "track",
       "road",
+      "bus",
```

Comment:
Is there any case where well tagged and correctly tagged area needs rendering `area:highway=bus`?

Reply:
> Areas for lanes (e.g. area:highway=bus and area:highway=shoulder) should be drawn inside the area of the highway they belong to. Parallel ways should each get their own area (e.g. dual carriageways, cycleways if drawn as separate ways). 

Looks like it would be covered by the underlying area:highway=* tag.